apps/fulfillment-e2e/src/integration/fully-picking-a-picklist.cy.ts

```diff
@@ -0,0 +1,45 @@
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+describe('Fully pick a picking list', () => {
+  const pickingListId = '37cb241e-f18a-5768-985c-a2d7aff4875e';
```

Comment:
@tarashaliuk added a nice method to get an ID and not hardcode it in the tests - here https://github.com/spryker/oryx/pull/280 - can we merge that PR in dev (it is approved, so everyone can do that) and use it here?

Reply:
Currently we can't take any id on the picking page(like theres is on picking lists items), because we still don't have the header information. 
As soon we have that, we use the id information on the header and not been needed to mock it on the e2e test